apps/services/views.py

```
@@ -0,0 +1,92 @@
+"""ViewSets for managing services."""
+from typing import ClassVar, List, Type
```

Ruff (0.14.0)

`typing.List` is deprecated, use `list` instead

(UP035)